dev_env/settings.py

```diff
@@ -42,12 +42,15 @@
         'simple': {
             'format': '%(levelname)s %(asctime)s %(message)s'
         },
+        'message_only': {
+            'format': '%(message)s'
```

Comment:
Is this formatter required?

Reply:
Yes, otherwise the original tests won't work (since they expect the logged message to equal the provided log message without any additional meta information).

Reply:
I'm fine with that :+1: